src/reduxTypes.js

```diff
@@ -434,6 +434,8 @@ export type PerAccountState = $ReadOnly<{
   ...
 }>;
 
+export opaque type PerAccountState: PerAccountStateImpl = PerAccountStateImpl;
```

Comment:
```
redux types: Stop letting GlobalState pose as PerAccountState
```

In particular (and I'm not sure if this needs a mention in the commit, I just want to check my understanding), this prevents GlobalState from posing as a PerAccountState _in code outside this file_. We totally can do that within the file, and in fact we do:

```js
// For now, under our single-active-account model, we want a GlobalState
// to be seamlessly usable as a PerAccountState.
(s: GlobalState): PerAccountState => s; // eslint-disable-line no-unused-expressions
```

Reply:
Yes, that's right. Mmm, and I should update that comment and/or cast, too, because they become misleading with this change.